### ISSUE TITLE
Training of FLAMO

### ISSUE THREAD

**Issue by SebastianJiroSchlecht** [artificial-audio/pyFDN#177]
Build convenient Ecosystem
- Initalizations
- Loss functions
- Training Pipeline

Compare with Orchi's and Jeremy's training scripts.
